core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -708,12 +709,24 @@ private void processWorkerAddedBlocks(MasterWorkerInfo workerInfo,
             mLostBlocks.remove(blockId);
           }
         } else {
-          LOG.warn("Failed to register workerId: {} to blockId: {}", workerInfo.getId(), blockId);
+          LOG.warn("Invalid block: {} from worker {}.", blockId,
+              workerInfo.getWorkerAddress().getHost());
         }
       }
     }
   }
 
+  @GuardedBy("workerInfo")
+  private void processWorkerOrphanedBlocks(MasterWorkerInfo workerInfo) {
+    for (long block: workerInfo.getBlocks()) {
```

Comment:
NIT: should there be a space before `:`?

Reply:
Fixed, thanks.